### ISSUE TITLE
Config files incorrect after Genie.newapp("MyGenieApp") and crash following bin/repl

### ISSUE THREAD

**Issue by martynjriley** [GenieFramework/Genie.jl#233]
MyGenieApp/config/env/dev.jl
MyGenieApp/config/env/test.jl

It seems both the above config files may be wrong regarding ...

"websocket_server" in the settings should be "websockets_server".

You get a crash when restarting the app with bin/repl without this change.

**Comment by essenciary** [comment]
@martynjriley It doesn't look like it. Can you please explain how I can reproduce the error and what version of Genie are you using?

**Comment by martynjriley** [comment]
I am using julia 1.3.1 and genie 0.31.0 on ubuntu 20.04.
I have just repeated the operations to create a new app with ...
>julia 
julia >using Genie
julia >Genie.newapp("MyGenieApp1")
.... All working
julia >exit()
> cd MyGenieApp1
> bin/repl
julia >up()
... All working

I am rather confused because that isn't what happened when I created the first app. Not sure what I did differently! If I look at dev.jl and test.jl in this second app then they have no final entry for websocket_server. In my first app this previously had an entry for websockets_server which I had to change to websocket_server to prevent the crash. I may have done something different the first time but if so, I have no idea what it might be!
